src/task/dto/create-task.dto.ts

```diff
@@ -1,23 +1,29 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsEnum, IsString, MinLength } from 'class-validator';
+import { TaskSeverity, TaskStatus } from '../../common';
+
 export class CreateTaskDto {
+  @ApiProperty({ example: 'Сесть за работу' })
+  @IsString()
+  @MinLength(3)
   title: string;
+
+  @ApiProperty({
+    example: 'Перестать прокастинировать и начать заниматься делом',
+  })
+  @IsString()
+  @MinLength(10)
   description: string;
+
+  @ApiProperty({ enum: TaskStatus, example: TaskStatus.NEW })
+  @IsEnum(TaskStatus)
   status: TaskStatus;
-  severity: TaskSeverity;
-  author: string;
-  assignee: string;
-}
 
-export enum TaskStatus {
-  NEW,
-  IN_PROGRESS,
-  ON_HOLD,
-  PENDING,
-  COMPLETED,
-}
+  @ApiProperty({ enum: TaskSeverity, example: TaskSeverity.HIGH })
+  @IsEnum(TaskSeverity)
+  severity: TaskSeverity;
 
-export enum TaskSeverity {
-  LOW,
-  MEDIUM,
-  HIGH,
-  CRITICAL,
+  @ApiProperty({ example: 'qdwdq-dasda-asda-dasda' })
+  @IsString()
+  assignee: string;
```

Comment:
```suggestion
  assigneeId: number;
```
Это на будущее - id юзера-исполнителя